samples/Solidity/RefundVault.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity ^0.4.11;
```

Comment:
The Travis CI build fails because of these samples. Files in `samples/` are used by the Bayesian classifer and are expected to be registered in `languages.yml` (through their file extension or filename). Since it's not the case here, could you move these example files to `tests/fixtures/`?